kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -322,6 +323,11 @@ def cargo_build(
         additional_args.append(crate)
     if mode == "release":
         additional_args.append("--release")
+    if features:
```

Comment:
out of interest, why are you seeking to enable these features in Kraken, rather than making them be default features in the root project. Basically, what's the use case for this functionality?

Reply:
@morosanmihail, I am keen to understand the use case as well :)

Reply:
Capturing for posterity: this would be useful to enable platform-specific features ([a workaround](https://github.com/rust-lang/cargo/issues/1197#issuecomment-1641086954) exists, but it's not great), as well as to have multiple cargo build tasks to produce variants of a binary, with different features combinations.